edge-fqm/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,88 @@
+function fn() {
+
+  karate.configure('logPrettyRequest', true);
+  karate.configure('logPrettyResponse', true);
+
+  var retryConfig = { count: 20, interval: 30000 }
+  karate.configure('retry', retryConfig)
+
+  var env = karate.env;
+
+  // The "testTenant" property could be specified during test runs
+  var testTenant = karate.properties['testTenant'];
+
+  var config = {
+    baseUrl: 'http://localhost:9130',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
+    prototypeTenant: 'diku',
+
+    testTenant: testTenant ? testTenant: 'testTenant',
+    testAdmin: {tenant: testTenant, name: 'test-admin', password: 'admin'},
+    testUser: {tenant: testTenant, name: 'test-user', password: 'test'},
+
+    // define global features
+    login: karate.read('classpath:common/login.feature'),
+    dev: karate.read('classpath:common/dev.feature'),
+
+    // define global functions
+    uuid: function () {
+      return java.util.UUID.randomUUID() + ''
+    },
+
+    random: function (max) {
+      return Math.floor(Math.random() * max)
+    },
+
+    randomMillis: function() {
+      return java.lang.System.currentTimeMillis() + '';
+    },
+
+    random_string: function() {
+      var text = "";
+      var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+      for (var i = 0; i < 5; i++)
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+      return text;
+    },
+
+    pause: function(millis) {
+          var Thread = Java.type('java.lang.Thread');
+          Thread.sleep(millis);
+    }
+  };
+
+  // Create 100 functions for uuid generation
+  var rand = function(i) {
+    karate.set("uuid"+i, function() {
+      return java.util.UUID.randomUUID() + '';
+    });
+  }
+  karate.repeat(100, rand);
+
+  if (env == 'snapshot-2') {
+    config.baseUrl = 'https://folio-snapshot-2-okapi.dev.folio.org:443';
+    config.edgeUrl = 'https://folio-snapshot-2.dev.folio.org:8000';
+    config.apikey = 'eyJzIjoiNXNlNGdnbXk1TiIsInQiOiJkaWt1IiwidSI6ImRpa3UifQ==';
```

Comment:
Nit: Maybe add a comment for the API key to say what the username and tenant are